src/Pie.js

```diff
@@ -69,18 +72,18 @@ export default class PieChart extends Component {
     let radius = Math.min(x, y)
 
     let chart = Pie({
-      center: this.props.center || (this.props.options && this.props.options.center) || [0,0] ,
-      r: this.props.r || (this.props.options && this.props.options.r) || radius /2,
-      R: this.props.R || (this.props.options && this.props.options.R) || radius,
+      center: this.props.options.center || [0,0],
+      r: this.props.options.r || radius /2,
```

Comment:
This has reverted a latest master change. The r value should still be pulled from either props.r or props.options.r as in current master.